Wiki/zh-Tw.js

```diff
@@ -0,0 +1,678 @@
+exports.LANGUAGE = {
+  CODE: "zh-Tw",
+  STANDARD_CODE: "zh-Tw",
+  FULL_CODE: "正體中文",
+  WIKI_CODE: "zh-Tw",
+  CONTRIBUTORS: ["Kami-Sqmf", "iLxlo_"]
+};
+
+exports.TIME = {
+  SECOND: "秒",
+  SECONDS: "秒",
+  MINUTE: "分",
+  MINUTES: "分",
+  HOUR: "小時",
+  HOURS: "小時",
+  DAY: "天",
+  DAYS: "天",
+  WEEK: "週",
+  WEEKS: "週",
+  MONTH: "月",
+  MONTHS: "月",
+  YEAR: "年",
+  YEARS: "年"
+};
+
+exports.PAGE_TITLES = {
+  HOME: "首頁",
+  INVITE_FILO: "邀請 Filo",
+  ANTI_BAD_WORDS: "髒話過濾",
+  ANTI_EVASION: "處罰者破解偵測",
+  ACTION_LIST: "事項清單",
+  ANTI_INVITES: "邀請偵測",
+  ANTI_MASS_MENTIONS: "大量提及偵測",
+  ANTI_REPEATED_TEXT: "重複文字偵測",
+  AUTO_MODERATION: "前科制度",
+  ANTI_SPAM: "洗版偵測",
+  COMMANDS: "指令",
+  DISABLE_COMMANDS: "停用指令",
+  FAREWELLS: "離開訊息",
+  FAREWELLS_VARIABLES: "離開訊息變數",
+  GUILD: "伺服器",
+  GUILD_INVITE: "伺服器邀請連結",
+  MUTED_ROLE: "禁言身分組",
+  LANGUAGE: "語言",
+  SUPPORTED_LANGUAGES: "支援的語言",
+  LOGGING: "日誌",
+  MEMBER_COUNTER: "人數統計 (頻道)",
+  MEMBER_COUNTER_NICKNAME: "人數統計 (暱稱)",
+  PREFIX: "Prefix",
+  REPORTS: "舉報",
+  WELCOMES: "歡迎訊息",
+  WELCOMES_VARIABLES: "歡迎訊息變數",
+  WELCOME_ROLE: "預設身分組",
+  CLOUDFLARE: "Cloudflare",
+  COMMON_ERRORS: "常見問題"
```

Comment:
I don't know if this is wrongly translated, if it is well translated you can leave it as it was (the translator does not translate it to me as "Frequent questions").

Must be "Common Errors".